zcash_client_sqlite/src/wallet.rs

```diff
@@ -1591,7 +1591,7 @@ pub(crate) fn account_birthday(
     conn.query_row(
         "SELECT birthday_height
          FROM accounts
-         WHERE account = :account_id",
+         WHERE id = :account_id",
```

Comment:
The account ID refactor, and the large-scale column name overhauling that was performed as a side-effect, has caused multiple of these bugs. Some of them we noticed and fixed in a hurry before 1.0, but like this one here, I do not doubt there are more.

What can we do to systematically check and fix all the queries, and ensure we don't have this problem in future? Defining a bunch of column name constants becomes unwieldy when it forces us to use string formatting, but if we start building wrappers to make that easier, we'd basically be building a custom DAO.

Reply:
Basically this just points to the fact that we actually need tests for all of the SQL queries; given that, such bugs become shallow.

Reply:
Is there some automated tool to lint / statically analyse SQL queries given a dump of the schema? It seems feasible.